controllers/app.controller.js

```diff
@@ -22,5 +22,13 @@ function articleController(request, response, next) {
     }) 
 }
 
-module.exports = {topicsController, apiController, articleController}
+function getAllArticles(request, response, next) {
+    return getAllArticlesModel().then(data => {
+        response.status(200).send({articles : data});
+    }).catch(err => {
+        next(err);
+    })
+}
```

Comment:
Well implemented, and good work sending the response back on a key of `articles`